lib/clientBuilder.js

```diff
@@ -99,6 +103,7 @@ function OktaAuthBuilder(args) {
     parseFromUrl: util.bind(token.parseFromUrl, sdk, sdk),
     decode: util.bind(token.decodeToken, sdk),
     refresh: util.bind(token.refreshToken, sdk, sdk),
+    verify: util.bind(token.verifyIdToken, sdk, sdk),
```

Comment:
It's a little strange, because we're only able to validate idTokens for now. Once we add support for verifying other tokens (via the introspection endpoint), then it may make sense to change it.

On the other hand, now would be the time to change the response if we want. Now, verifyIdToken returns `true` or `false`, but it doesn't jive with how we validate claims. In validateClaims, we throw errors when validation fails, with descriptions of the failure. It'd be nice if I changed this to throw descriptive errors on validation failure.